eng/common/pipelines/templates/jobs/spec-gen-sdk.yml

```diff
@@ -1,28 +1,40 @@
 parameters:
-  - name: SpecRepoUrl
-    type: string
-  - name: SpecRepoBranch
-    type: string
   - name: SpecRepoCommit
     type: string
-  - name: SdkRepoBranch
+    default: 'HEAD'
+  - name: SdkRepoCommit
     type: string
+    default: 'HEAD'
   - name: ConfigType
     type: string
-  - name: TspConfigPath
-    type: string
-  - name: ReadmePath
+    values:
+      - 'TypeSpec'
+      - 'OpenAPI'
+    default: 'TypeSpec'
+    displayName: 'Configuration Type'
+  - name: ConfigPath
     type: string
+    default: 'specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml'
+    displayName: 'Path to TypeSpec or OpenAPI Configuration file'
   - name: ApiVersion
     type: string
+    default: 'not-specified'
+    displayName: 'API Version of Spec to be used for SDK Generation'
   - name: SkipPullRequestCreation
     type: boolean
+    default: false
+
+variables:
```

Comment:
I would make this a template parameter instead of a variable. You should still default it to the pipeline variable you can simply call it `$(SdkRepoUrl)`